ruby/lib/extism/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Extism
-  VERSION = "0.0.1.rc4"
+  VERSION = "0.0.1.rc6"
```

Comment:
Is it an Elixir specific thing to exclude the `.` between rc and the number?

I see we already released it without last time - so nbd, since we'll be off this version soon enough. Just curious.

Reply:
https://guides.rubygems.org/patterns/

it's how they recommend it. Either way may work though. Im not sure